library/psa_crypto.c

```diff
@@ -4948,8 +4948,13 @@ static psa_status_t psa_key_agreement_internal(psa_key_derivation_operation_t *o
                                                size_t peer_key_length)
 {
     psa_status_t status;
+#if PSA_KEY_AGREEMENT_MAX_SHARED_SECRET_SIZE != 0
```

Comment:
As with https://github.com/Mbed-TLS/mbedtls/pull/8079#discussion_r1296993163, this shouldn't be needed since `PSA_KEY_AGREEMENT_MAX_SHARED_SECRET_SIZE` is `MBEDTLS_ECP_MAX_BYTES` which is ≥1.

Reply:
Done.